docs/csharp/language-reference/statements/checked-and-unchecked.md

```diff
@@ -0,0 +1,65 @@
+
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/checked-and-unchecked/Program.cs" id="OperatorForm":::
+
+The `checked` and `unchecked` statements and operators only affect the overflow-checking context for those operations that are *textually* inside the statement block or operator's parentheses, as the following example shows:
+
+:::code language="csharp" interactive="try-dotnet-method" source="snippets/checked-and-unchecked/Program.cs" id="ScopeExample":::
+
+At the preceding example, the first invocation of the `Multiply` local function shows that the `checked` statement doesn't affect the overflow-checking context within the `Multiply` function as no exception is thrown. At the second invocation of the `Multiply` function, the expression that calculates the second argument of the function is evaluated in a checked context and results in an exception as it's textually inside the block of the `checked` statement.
+
+## Operations affected by the overflow-checking context
+
+The overflow-checking context affects the following operations:
+
+- The following built-in [arithmetic operators](../operators/arithmetic-operators.md): unary `++`, `--`, `-` and binary `+`, `-`, `*`, and `/` operators, when their operands are of integral type (that is, either [integral numeric](../builtin-types/integral-numeric-types.md) or [char](../builtin-types/char.md) type) or [enum](../builtin-types/enum.md) type.
+- [Explicit numeric conversions](../builtin-types/numeric-conversions.md#explicit-numeric-conversions) between integral types or from [`float` or `double`](../builtin-types/floating-point-numeric-types.md) to an integral type.
+
+  > [!NOTE]
+  > When you convert a `decimal` value to an integral type and the result is outside the range of the destination type, an <xref:System.OverflowException> is always thrown, regardless of the overflow-checking context.
+
+- Beginning with C# 11, user-defined checked operators and conversions. For more information, see the [User-defined checked operators](../operators/arithmetic-operators.md#user-defined-checked-operators) section of the [Arithmetic operators](../operators/arithmetic-operators.md) article.
+
+## Default overflow-checking context
+
+If you don't specify the overflow-checking context, the value of the [**CheckForOverflowUnderflow**](../compiler-options/language.md#checkforoverflowunderflow) compiler option defines the default context for non-constant expressions. By default the value of that option is unset and integral-type arithmetic operations and conversions are executed in an **unchecked** context.
+
+Constant expressions are evaluated by default in a checked context and a compile-time error occurs in the case of an overflow. You can explicitly specify an unchecked context for a constant expression with the `unchecked` statement or operator.
```

Comment:
IIRC, this behavior didn't change with the new checked and unchecked operators. Tagging @tannergooding to confirm. (If my memory is correct, this is fine as written)

Reply:
Yep. C# has always evaluated `const T = expr` as if it were checked and want's you to explicitly `uncheck(expr)` in the case it would overflow.